install.sh

```diff
@@ -68,6 +67,20 @@ patch () {
 
     pass=$(sha256sum $APP_BINARY | cut -c1-64)
     wget -O- $PATCH_URL/${patch_version}_$hash.patch | openssl aes-256-cbc -d -a -md sha512 -pbkdf2 -iter 1000000 -salt -pass pass:$pass | tar --overwrite -xjC $CACHE_DIR
+    cat << EOF > /etc/systemd/system/xochitl.service.d/override-onfailure.conf
```

Comment:
I could not get this to work. it kept softbricking my device. I tried a few things.
-  Firstly, I notice that you are putting OnFailure under [Service] and not [Unit]. However adjusting for this did not help
- Secondly, you are calling a bash script instead of specifying another service. I tried adding a drop in at `/etc/systemd/system/remarkable-fail.d/override-onfailure.conf`  and made `ExecStart=/home/root/.local/bin/rm-hacks-remarkable-fail.sh`. This also did not work. I don't know why. 
- Finally, rather than use an Extended drop in as above. I created an override, I copied `/lib/system/systemd/remarkable-fail.service` to `/etc/systemd/system/remarkable-fail.service` and modified the ExecStart line to `/home/root/.local/bin/rm-hacks-remarkable-fail.sh` 

So i would recommend adding following to `/etc/systemd/system/remarkable-fail.service`
`[Unit]
Description=reMarkable fail application

[Service]
ExecStart=/home/root/.local/bin/rm-hacks-remarkable-fail.sh
Restart=on-failure
`

Reply:
This is what I get for trying to develop something without my device to double-check anything :face_with_spiral_eyes: 
The override needs to be for remarkable-fail instead of xochitl as you've discovered.

Reply:
I've elected to still use drop-in replacements, as this is the preferred mechanism for this kind of thing that we've been using in the community. It makes it easier to layer multiple things on top of each other. Completely replacing service files can cause other unintended side effects when toltec eventually catches up.

Reply:
realizing this works just fine. When testing before I just forgot to touch `/tmp/rm-hacks-skip-onfailure`.